From e56c7ba39cad0724faaacfacea6473f775f7b376 Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Wed, 19 Jan 2011 18:36:07 +0800
Subject: ENGR00138201 MX53_SMD: BT: add delay after BT chip reset

add 100ms delay after BT chip reset, make it work stable.

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 arch/arm/mach-mx5/board-mx53_smd.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/mach-mx5/board-mx53_smd.c b/arch/arm/mach-mx5/board-mx53_smd.c
index eb2357e..7fcb794 100755
--- a/arch/arm/mach-mx5/board-mx53_smd.c
+++ b/arch/arm/mach-mx5/board-mx53_smd.c
@@ -523,6 +523,8 @@ static void mx53_smd_bt_reset(void)
 	/* pull up after power supply BT */
 	gpio_set_value(MX53_SMD_BT_RESET, 1);
 	gpio_free(MX53_SMD_BT_RESET);
+	msleep(100);
+	/* Bluetooth need some time to reset */
 }
 
 static int mx53_smd_bt_power_change(int status)
-- 
1.7.9.5

